bioio_ome_zarr/reader.py

```diff
@@ -36,6 +36,7 @@ class Reader(reader.Reader):
     _metadata: Optional[Any] = None
     _scenes: Optional[Tuple[str, ...]] = None
     _current_scene_index: int = 0
+    _resolution_level_dict: Optional[Dict[int, Tuple[int, ...]]] = None
```

Comment:
can remove this?

Reply:
yeah, unfortunately...

Reply:
so all readers need it??

Reply:
no I stand corrected! hopping around too many repos... resolved in [fb3e575](https://github.com/bioio-devs/bioio-ome-zarr/pull/23/commits/fb3e57531c7bcca8671f4852e4a21d561f6ffd56)